weave

```diff
@@ -847,7 +849,7 @@ case "$COMMAND" in
             -p $PORT:$CONTAINER_PORT/tcp -p $PORT:$CONTAINER_PORT/udp \
             -v /var/run/docker.sock:/var/run/docker.sock \
             -e WEAVE_PASSWORD \
-            $WEAVE_DOCKER_ARGS $IMAGE -iface $CONTAINER_IFNAME -port $CONTAINER_PORT -name "$PEERNAME" -nickname "$(hostname)" $IPRANGE "$@")
+            $WEAVE_DOCKER_ARGS $IMAGE -iface $CONTAINER_IFNAME -port $CONTAINER_PORT -name "$PEERNAME" -nickname "$(hostname)" $IPRANGE $ARGS)
```

Comment:
I tried a few things but fundamentally `"$@"` is very special and I can't see a way to synthesise something similar using regular shell variables.

If the case-statement did a 'break' as before, but covered every expected parameter, then the original problem wouldn't arise.  But this is rather fragile.